packages/components/src/FilterDateRangePicker/subcomponents/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react"
+import { FieldMessage } from "@kaizen/draft-form"
+import { DateRangeFieldValidationMessage } from "../FilterDateRangePickerField/types"
+import styles from "./DateRangeValidationMessage.module.scss"
+
+export interface DateRangeValidationMessageProps {
+  validationMessage?: DateRangeFieldValidationMessage
+  dateStartId?: string
+  dateEndId?: string
+  isReversed?: boolean
+}
+
+export const DateRangeValidationMessage = ({
+  validationMessage,
+  dateStartId,
+  dateEndId,
+  isReversed,
+}: DateRangeValidationMessageProps): JSX.Element | null => {
+  if (!validationMessage) return null
+
+  const dateStart = validationMessage.dateStart
+  const dateEnd = validationMessage.dateEnd
+
+  if (dateStart && dateEnd) {
+    if (dateStart.status === dateEnd.status) {
+      return (
+        <FieldMessage
+          message={
+            <ul className={styles.fieldMessageList}>
+              <li id={dateStartId}>{dateStart.message}</li>
+              <li id={dateEndId}>{dateEnd.message}</li>
+            </ul>
+          }
```

Comment:
Out of scope, but maybe `message` could accept an array and pump this out for us.

Reply:
Although the `id` would be harder to do

Reply:
Definitely more for a generic component to handle, but not right now 😉